airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -260,14 +262,16 @@ public void failAttempt(long jobId, int attemptNumber) throws IOException {
   @Override
   public void succeedAttempt(long jobId, int attemptNumber) throws IOException {
     final LocalDateTime now = LocalDateTime.ofInstant(timeSupplier.get(), ZoneOffset.UTC);
+    final LocalDateTime endedAtTruncatedToSeconds = LocalDateTime.ofEpochSecond(now.toEpochSecond(ZoneOffset.UTC), 0, ZoneOffset.UTC);
     database.transaction(ctx -> {
       // override any other terminal statuses if we are now succeeded.
       updateJobStatus(ctx, jobId, JobStatus.SUCCEEDED, now);
 
       ctx.execute(
-          "UPDATE attempts SET status = CAST(? as ATTEMPT_STATUS), updated_at = ? WHERE job_id = ? AND attempt_number = ?",
+          "UPDATE attempts SET status = CAST(? as ATTEMPT_STATUS), updated_at = ? , ended_at = ? WHERE job_id = ? AND attempt_number = ?",
```

Comment:
can you create an issue to make sure this is standardised?

Reply:
😬 damn. so we previously haven't been setting `ended_at`? good catch.